DESCRIPTION

```diff
@@ -143,6 +143,7 @@ Collate:
     'redundant_ifelse_linter.R'
     'regex_subset_linter.R'
     'routine_registration_linter.R'
+    'rprojroot_utils.R'
```

Comment:
not a big fan of the name (too focused on .Rproj)... how about 'package_metadata_utils.R'?

Reply:
Done!

I had named it so originally because most of these helpers functions were taken from `{rprojroot}` package, but, yeah, we don't necessarily need that to be reflected in the file name.